docs/Installing-Clear-Containers-3.0-on-Fedora.md

```diff
@@ -0,0 +1,54 @@
+# Installing Clear Containers 3.0 Alpha on Fedora
+
+## Introduction
+
+Clear Containers **3.0-alpha** is available for Fedora\* versions **24** and **25**.
+
+## Install latest Docker
```

Comment:
Do not use headings for steps. Please use ordered lists:
1. Install the latest version of Docker using the following commands: